apis_core/apis_entities/edit_generic.py

```diff
@@ -49,8 +80,7 @@ def get(self, request, *args, **kwargs):
             .select_subclasses()
         )
 
-        for entity_class in caching.get_all_entity_classes():
-
+        for entity_class in get_classes_with_allowed_relation_from(kwargs["entity"]):
```

Comment:
The View is using the `EntityInstanceMixin`, so you can access the entity name by using `self.entity` instead of kwargs - that way we have one single point of access for that value.

Reply:
Didn't realise that... I thought using `kwargs` was a bit ugly!